modules/gitops-argocd-projects.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argocd-projects_{context}"]
+= Argo CD projects
+
+Argo CD projects, not to be confused with {OCP} projects, provide a way to group applications together. With Argo CD projects, you can allow restrictions by placing them on the Applications concerning what resources can be deployed and where they can be deployed. Additionally, you can enable Argo CD role-based access control (RBAC) controls by defining them more granularly at the project level versus the global level in the Argo CD custom resource (CR).
+
+While you can define tenant RBAC globally in the Operator’s Argo CD CR, you must define tenant RBAC along with restrictions in Argo CD `AppProject` CR.
+
+In scenarios where there are a significant number of tenants, attempting to manage all tenants with global RBAC might quickly become cumbersome. 
```

Comment:
Addressed.

Reply:
@jannfis and @gnunn1: Sending this gentle reminder and requesting you to provide me with inputs from the discussion in this thread. As the release date closes, it would be great to accommodate inputs if the information is included in GitOps 1.15.  The release date is December 11th.

Thanks!